src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-button-strategy.ts

```diff
@@ -32,11 +32,11 @@ export default class PaypalCommerceButtonStrategy implements CheckoutButtonStrat
                 const config = state.config.getStoreConfig();
                 const cart = state.cart.getCart();
 
-                if (!config) {
+                if (!config || !config.currency) {
```

Comment:
I think `currency` is not an optional property so it's okay for you to not do a null check here.

Reply:
Okay. Thank you!